src/server.js

```diff
@@ -100,6 +113,14 @@ in ${config.public ? "public" : "private"} mode`);
 
 		manager.autoloadUsers();
 	}
+
+	function shutdown() {
+		log.info("Quitting...");
+		server.close(); // Removes socket file
+		process.exit();
+	}
+
+	process.on("SIGINT", shutdown);
```

Comment:
What are the consuqences of this? I know hooking this might introduce weirdness because it disables some internal nodejs handling.